src/modules/pageDataGroup.js

```diff
@@ -0,0 +1,23 @@
+import DataGroups from './dataGroup.js';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const PageDataGroup = (props) => {
+    return (
+        <DataGroups
+            {...props}
+            moduleType="page"
+        />
+    );
+};
+
+PageDataGroup.propTypes = {
+    bleed: PropTypes.bool,
+    className: PropTypes.string,
+    data: PropTypes.object.isRequired,
+    header: PropTypes.string,
+    rows: PropTypes.array.isRequired,
```

Comment:
Let's change this `PropType` `array to `arrayOf(shape)` and define the types for those propps as well.

Reply:
Fixed